.gitignore

```
@@ -134,3 +134,8 @@ android-secret.json
 
 # Fonts
 android/app/src/main/assets/fonts
+
+
+# Firebase files
+android/app/google-services.json
+GoogleService-Info.plist
```

```suggestion
GoogleService-Info.plist

```
Just a friendly newline :D